redisq/queue.go

```diff
@@ -329,6 +336,25 @@ func (q *Queue) scheduleDelayed(ctx context.Context) (int, error) {
 	return len(bodies), nil
 }
 
+func (q *Queue) scheduleCleanZombieConsumers(ctx context.Context) (int, error) {
+	consumers, err := q.redis.XInfoConsumers(ctx, q.stream, q.streamGroup).Result()
+
+	if err != nil {
+		return 0, err
+	}
+
+	for _, consumer := range consumers {
+		if consumer.Name == q.streamConsumer {
```

Comment:
Would be nice to add a comment for future readers why you skip this.

Reply:
done

Reply:
I guess it is not really needed and is just an optimization? Just checking :)

Reply:
adding this line , because I am not sure what will happen if we delete the current consumer, just in case.